### DISCUSSION TITLE
Integrating Translation capabilities

### DISCUSSION THREAD

**Discussion by dptsup** [chidiwilliams/buzz#645] [Ideas]
I attempted to translate text into a different language using Whisper, but since it only supports English translation, I found DesktopTranslator as a solution. Would it be feasible to incorporate it as a tool or feature into your software?

[https://github.com/ymoslem/DesktopTranslator](url)

**Comment by raivisdejus** [comment]
Latest versions of Buzz also support translations using AI. For usage see https://chidiwilliams.github.io/buzz/docs/usage/translations
